src/layout/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+import { Suspense } from 'react';
+import { Outlet } from 'react-router';
+
+import Header from './Header';
+import Loader from './Loader';
+import { Container } from '../components/ui/Container';
+
+const SystemLayout: FC = () => {
```

Comment:
The component name should correspond to the folder name.

Reply:
done